madgraph5amcatnlo-toolfile.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external madgraph5amcatnlo-toolfile 1.0
+Requires: madgraph5amcatnlo
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/madgraph5amcatnlo.xml
+<tool name="madgraph5amcatnlo" version="@TOOL_VERSION@">
+  <lib name="MadGraph5aMCatNLO"/>
+  <client>
+    <environment name="MADGRAPH5AMCATNLO_BASE" default="@TOOL_ROOT@"/>
+  </client>
+  <runtime name="ROOT_INCLUDE_PATH" value="$MADGRAPH5AMCATNLO_BASE" type="path"/>
+  <runtime name="ROOT_PATH" value="$MADGRAPH5AMCATNLO_BASE" type="path"/>
```

Comment:
No, we don't want this in the path, because we still commonly use standalone madgraph in a cmssw environment, and want to avoid conflicts.